android/app/src/main/java/com/zulipmobile/MainActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.zulipmobile
+
+import android.os.Bundle
+import android.webkit.WebView
+import com.facebook.react.ReactActivity
+
+open class MainActivity : ReactActivity() {
+    /**
+     * Returns the name of the main component registered from JavaScript.
+     * This is used to schedule rendering of the component.
+     */
+    override fun getMainComponentName(): String? {
```

Comment:
Let's make it `String` -- that seems more true to the intent of the method.

(I'm guessing that the Android Studio auto-conversion went for `String?`, which is natural because Java doesn't make the distinction. The cleanup to `String` can be in a followup commit -- it's fine and good for that sort of cleanup to make the Kotlin more idiomatic to be in a separate commit from the conversion.)